tests/framework/extensions/clusters/kubernetesversions/default.go

```diff
@@ -24,9 +24,10 @@ func Default(client *rancher.Client, provider string, kubernetesVersions []strin
 		if kubernetesVersions == nil {
```

Comment:
default_version should be defaultVersion

Reply:
fixed